delphin/cli/edm.py

```diff
@@ -56,14 +70,14 @@ def _iter_representations(path: Path, fmt: str, p: int):
             except IndexError:
                 yield None
             else:
-                yield from_mrs(result.mrs(), predicate_modifiers=True)
+                yield eds_from_mrs(result.mrs(), predicate_modifiers=True)
 
     elif path.is_file():
         codec = util.import_codec(fmt)
         rep = codec.CODEC_INFO.get('representation', '').lower()
         if rep == 'mrs':
             for mrs in codec.load(path):
-                yield from_mrs(mrs, predicate_modifiers=True)
+                yield eds_from_mrs(mrs, predicate_modifiers=True)
```

Comment:
Now that `delphin.mrs` is imported, it would be less confusing to rename this variable to `m` or `sr` (semantic representation).